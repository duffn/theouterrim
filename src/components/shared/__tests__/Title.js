import React from "react"
import renderer from "react-test-renderer"
import Title from "../Title"

describe("Title", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Title>The Outer Rim</Title>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
