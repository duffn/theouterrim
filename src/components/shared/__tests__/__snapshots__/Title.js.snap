// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<h1
  className="MuiTypography-root MuiTypography-h4 MuiTypography-colorPrimary MuiTypography-gutterBottom"
  style={
    Object {
      "fontFamily": "Saira Semi Condensed",
    }
  }
>
  The Outer Rim
</h1>
`;
